modules/csharp/pages/gs-install.adoc

```diff
@@ -58,10 +68,38 @@ This is the recommended method of dependency management because it supports the
 * `Couchbase.Lite.Support.NetDesktop`
 * `Couchbase.Lite.Support.WinUI`
 
+=== Installing Vector Search 
+
+Couchbase recommends installing Nuget packages via _PackageReference_.
+
+[IMPORTANT]
+--
+To use Vector Search, you must have Couchbase Lite installed and add the Vector Search extension to your Couchbase Lite application.
+Vector Search is available only for 64-bit architectures and
+Intel processors that support the Advanced Vector Extensions 2 (AVX2) instruction set.
+To verify whether your device supports the AVX2 instructions set, https://www.intel.com/content/www/us/en/support/articles/000090473/processors/intel-core-processors.html[follow these instructions.] 
+--
+
+==== Package Reference
+This is the recommended method of dependency management because it supports the strict version requirement between the core _Couchbase Lite for .Net_ package and its dependent Support library, which comprises:
+
+.`Couchbase.Lite.Enterprise`
```

Comment:
Does this really need to be duplicated here?  Aren't these install instructions already somewhere else for the main package?  This is making this page overly verbose imo.

Reply:
I see your point. I've condensed to a couple of lines but will keep the section to make sure Vector Search instructions are distinguishable.